lib/msal-node/src/client/ManagedIdentitySources/BaseManagedIdentitySource.ts

```diff
@@ -84,6 +85,15 @@ export abstract class BaseManagedIdentitySource {
     ): ServerAuthorizationTokenResponse {
         let refreshIn, expiresIn: number | undefined;
         if (response.body.expires_on) {
+            // if the expires_on field in the response body is a string and in ISO 8601 format, convert it to a Unix timestamp (seconds since epoch)
+            if (
```

Comment:
nit: strange use of paranthesis. why not 

```
if (condition && condition) {
    do_stuff();
}
```

Reply:
? There's other logic already happening in this if statement:

// compute refresh_in as 1/2 of expires_in, but only if expires_in > 2h